core/src/main/scala/com/databricks/labs/remorph/generators/sql/ExpressionGenerator.scala

```diff
@@ -1,20 +1,29 @@
 package com.databricks.labs.remorph.generators.sql
 
-import com.databricks.labs.remorph.generators.GeneratorContext
-import com.databricks.labs.remorph.parsers.intermediate.Literal
+import com.databricks.labs.remorph.generators.{Generator, GeneratorContext}
+import com.databricks.labs.remorph.parsers.intermediate.{Expression, Literal}
 import com.databricks.labs.remorph.parsers.{intermediate => ir}
+import com.databricks.labs.remorph.transpilers.TranspileException
 
-import java.text.SimpleDateFormat
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, LocalDate, LocalDateTime, ZoneId, ZonedDateTime}
 import java.util.Locale
 
-class ExpressionGenerator {
-  private val dateFormat = new SimpleDateFormat("yyyy-MM-dd")
-  private val timeFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS")
+class ExpressionGenerator extends Generator[ir.Expression, String] {
+  private val dateFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd")
+  private val timeFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS")
```

Comment:
Won't this change still suffer from returning the default TZ and not UTC? The tests fail on my system because it is on Belize/Central America. 

```scala
private val timeFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS").withZone(ZoneId.of("UTC"))
```